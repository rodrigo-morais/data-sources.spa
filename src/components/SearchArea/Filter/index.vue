<template src='./Filter.html'></template>

<script src='./Filter.js'></script>

<style scoped src='./Filter.css'></style>
