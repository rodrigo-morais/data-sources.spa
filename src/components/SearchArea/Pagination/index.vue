<template src='./Pagination.html'></template>

<script src='./Pagination.js'></script>

<style scoped src='./Pagination.css'></style>
