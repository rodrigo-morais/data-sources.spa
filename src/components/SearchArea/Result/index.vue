<template src='./Result.html'></template>

<script src='./Result.js'></script>

<style scoped src='./Result.css'></style>
