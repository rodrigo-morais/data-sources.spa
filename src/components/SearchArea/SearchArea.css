section {
  display: grid;
  grid-template-columns: 1fr 1fr;
  grid-template-rows: 20% 80%;

  grid-column: 1;
  grid-row: 3;

  background-color: #F9F9F8;
}
