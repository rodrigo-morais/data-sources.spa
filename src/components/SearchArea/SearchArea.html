<section>
  <SearchFilter/>
  <SearchResult/>
  <SearchPagination />
</section>
