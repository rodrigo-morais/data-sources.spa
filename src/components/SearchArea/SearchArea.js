import Vue from 'vue';
import Component from 'vue-class-component';

export default Component({
  name: 'SearchArea',
})(class SearchArear extends Vue {});
