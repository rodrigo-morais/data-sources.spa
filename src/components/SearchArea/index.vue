<template src='./SearchArea.html'></template>

<script src='./SearchArea.js'></script>

<style scoped src='./SearchArea.css'></style>
