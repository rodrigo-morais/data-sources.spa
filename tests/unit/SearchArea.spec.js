import { shallowMount } from '@vue/test-utils';
import { createRenderer } from 'vue-server-renderer';
import SearchArea from '@/components/SearchArea/index.vue';

describe('SearchArea.vue', () => {
  it('renders according to design', () => {
    const renderer = createRenderer();
    const wrapper = shallowMount(SearchArea);
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err);
      expect(str).toMatchSnapshot();
    });
  });
});
