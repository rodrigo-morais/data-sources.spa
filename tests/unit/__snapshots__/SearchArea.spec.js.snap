// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchArea.vue renders according to design 1`] = `<section></section>`;
